packages/utils/package.json

```
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "bs58": "^4.0.1",
-    "eth-object": "aurora-is-near/eth-object#383b6ea68c7050bea4cab6950c1d5a7fa553e72b",
+    "eth-object": "aurora-is-near/eth-object#129c49bea44e66f0360c7312575511e86e1324ef",
```

I think, you are referring to the incorrect hash, this should target this commit: https://github.com/aurora-is-near/eth-object/commit/08131d3b69ce7863a0411c28915bbee75bba5725
```suggestion
    "eth-object": "aurora-is-near/eth-object#08131d3b69ce7863a0411c28915bbee75bba5725",
```

---

No, that commit had a bug which I fixed in https://github.com/aurora-is-near/eth-object/pull/13

---

Ok, the new commit which is in `master` is https://github.com/aurora-is-near/eth-object/commit/a84d1420fdde87a768b96c6a3b3d0ee435998f72